### DISCUSSION TITLE
Newsletter

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#174] [Ideas]
It's pretty common for websites and apps to have a newsletter.

I suggest we make a newsletter component that registers an email on the email service and add documentation around how to send emails to those subscribers for various use-cases.

**Comment by kentcdodds** [comment]
Agreed. Could you make an example of what you think that would look like? Then we can think about getting that into the main template.

**Reply by andrecasal** [reply]
I can eventually, but I'm working on the "getting started" checklist at the moment. Would be nice if someone else could take this up.

**Comment by andrecasal** [comment]
I've done a bit of research on email service providers for marketing:

| Name | Free contacts | Free sends | Cheapest tier | Contacts on this tier | Sends on this tier |
| ---|---|---|---|---|---|
| MailerLite		| 1000 |			12000				| $13 | 1000 |			Unlimited |
| MailChimp		| 500 |			1000 (500/day)	| $13 | 1000 |			5000 |
| MailJet			| Unlimited |	6000 (200/day)	| $15 | Unlimited |	50000 |
| OmniSend			| 250 |			500				| $16 | 500 |			6000 |
| ConvertKit		| 1000 |			Unlimited				| $25 | 1000 |			Unlimited |
| Brevo				| ? |				9000				| $25 | ? |				20000 |
| HubSpot			| ? |				2000				| $45 | 1000 |			5000 |

Feel free to add more providers and details.

**Reply by kentcdodds** [reply]
Oh, I definitely have already chosen the one that will recommend. It's convertkit. That's the one that I use. It's great.
